### ISSUE TITLE
Running Segmentation

### ISSUE THREAD

**Issue by frankvp11** [facebookresearch/dinov2#136]
I was able to figure out how to use the PCA's to get something of an image, however this isn't like most segmentation models, as it doesn't give mask(s). This is what I managed to get from the PCA:
![image](https://github.com/facebookresearch/dinov2/assets/90110502/56f42014-1fcc-467d-a49f-9a03404dde4e)
from following this repo: https://github.com/purnasai/Dino_V2
In this repo, I used the PCA visualization colab file.
Anyhow, to get to the issue at hand, it is my understanding that we must use a linear layer to get the semantic segmentation, a file of which has been provided for us (/eval/linear.py) however I have no idea how to use this file. 
Might someone by chance be willing/able to provide a tutorial/steps on how I can take an image (and for now pre-trained weights- would like custom in future) to get the segmentation using DINO?

**Comment by qasfb** [comment]
We do not provide semantic segmentation heads for DINOv2 at the moment; the eval/linear.py is for training a linear classifier for image classification (we use it for benchmarking Imagenet-1k classification results typically).

DINOv2 models only produce features; in order to perform segmentation, you would need to train a segmentation model using the DINOv2 pretrained backbones.

I encourage you to check out the mmseg library on this specific matter, as we have heard good things about it. https://github.com/open-mmlab/mmsegmentation/

**Comment by frankvp11** [comment]
Alright, I will be sure to check it out.
